doc/docs/master/how-tos/basic-data-operations/sql-ingest.md

```diff
@@ -66,20 +97,40 @@ Where:
 
 | Parameter     | Description | 
 | ------------- |-------------| 
-| **protocol**   | The name of the database protocol. As of today, you have the ability to connect to  `postgres`or `mysql`. Note that `mysql` allows you to connect to mariadb. |
+| **protocol**   | The name of the database protocol. <br> As of today, we support: <br>- `postgres` : connect to Postgresql or compatible (for example Redshift).<br>- `mysql` : connect to MySQL or compatible (for example MariaDB). <br>- `snowflake` : connect to Snowflake. |
 | **username**  | The user used to access the database.|
 | **host**      | The hostname of your database instance.|
 | **port**      | The port number your instance is listening on.|
 | **database**  | The name of the database to connect to. | 
 
+
+
+!!! Attention "Snowflake users, you will need a variant or the url above."
+     Pachyderm supports two connection URL patterns to query Snowflake: 
+
+      - `snowflake://username@<account_identifier>/<db_name>/<schema_name>?warehouse=<warehouse_name>`
+      - `snowflake://username@hostname:port/<db_name>/<schema_name>?account=<account_identifier>&warehouse=<warehouse_name>`
+
+     where:
+
+      - *[`account_identifier`](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html)* takes the following form for most URLs: `organization_name`-`account_name` .
+      - *`db_name`/`schema_name`* are respectfully the Database Name and the Schema (namespace) targeted.
+      - Additionally, a *[`warehouse`](https://docs.snowflake.com/en/user-guide/warehouses.html#virtual-warehouses)*, or “compute ressource” is required for all queries. Pass your warehouse as a parameter to the url: `warehouse=<warehouse_name>`
```

Comment:
I don't think warehouse is technically required. Also you got a typo in `compute ressource`

Reply:
You need a warehouse for all queries (as stated here https://docs.snowflake.com/en/user-guide/warehouses.html#virtual-warehouses). 
If none is provided, I am guessing that a default is used. In the case where you have multiple warehouses, which one would that be then? I can't find the proper documentation therefore resorted to always providing the targeted warehouse explicitly. 

Can you point me to the docs that describe what happens when no warehouse is explicitly mentioned?

Reply:
@albscui Do you have a the list of all the databases we support?

Reply:
Gotcha, I think it's safe to ensure the user specifies the warehouse then.

We currently support:
- postgres (and by extension Postgres derivatives like RedShift)
- mysql
- snowflake